invenio_communities/templates/semantic-ui/invenio_communities/frontpage.html

```diff
@@ -36,7 +37,7 @@ <h1 class="ui header">
         <form action="{{ url_for('invenio_communities.communities_search') }}" class="ui form">
           <div class="ui fluid action input">
             <input type="text" name="q" class="form-control" placeholder="{{ _('Search communities') }}">
-            <button type="submit" class="ui icon search button"><i class="search icon"></i></button>
+            <button type="submit" class="ui icon search button" aria-label="Search communities"><i class="search icon"></i></button>
```

Comment:
Oh just noticed this: at NU we can get away without translations, but for international partners it's probably nicer to have accessible label translated:

```suggestion
            <button type="submit" class="ui icon search button" aria-label="{{ _('Search communities'} }}"><i class="search icon"></i></button>
```

Or given the line is pretty long :sweat_smile: :

```suggestion
            <button 
              type="submit" 
              class="ui icon search button" 
              aria-label="{{ _('Search communities') }}"
            >
              <i class="search icon"></i>
            </button>
```